packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -175,7 +177,12 @@ const ffToStandardResourceTypeMap: { [ff: string]: ResourceType } = {
 }
 
 export class CdpAutomation {
-  constructor (private sendDebuggerCommandFn: SendDebuggerCommand, onFn: OnFn, private automation: Automation) {
+  constructor (
+    private sendDebuggerCommandFn: SendDebuggerCommand,
+    onFn: OnFn,
+    private automation: Automation,
+    private options: any,
```

Comment:
can you type this please, and maybe move `sendDebuggerCommandFn`, `onFn`, and `automation` to this object? let's keep this signature as clean as possible, it has been ballooning

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/20450/commits/4ed06ad6c835ae780ea8ed3354336c75cdea3ff8